apps/macos/src/Podfile.lock

```diff
@@ -490,6 +490,6 @@ SPEC CHECKSUMS:
   SwiftLint: 99f82d07b837b942dd563c668de129a03fc3fb52
   Yoga: 77d9987384e26b4aaf3e7a01808838cc2d4304f2
 
-PODFILE CHECKSUM: 15eda06aaa95b3f7091a3051b06322d719f25ff3
+PODFILE CHECKSUM: 2bf562c3be6637688b436c9b38272f364e9830f4
 
-COCOAPODS: 1.10.2
+COCOAPODS: 1.10.1
```

Comment:
nit: Looks like you should update your cocoapods installation.

Reply:
An aside on this: the version of cocoa pods from home-brew now works on m1 Macs without needing any rosetta aliasing! So I actually uninstalled the gem cocoa pods version and use the home-brew one instead now =)

Reply:
Fixed in commit 4378c5e17db2b6ca78558a9df1b64b040fe620a0